src/outputs/parthenon_hdf5.cpp

```diff
@@ -539,6 +539,8 @@ void PHDF5Output::WriteOutputFileImpl(Mesh *pm, ParameterInput *pin, SimTime *tm
                 vinfo.nx6 * vinfo.nx5 * vinfo.nx4 * vinfo.nx3 * vinfo.nx2 * vinfo.nx1;
           }
           memset(tmpData.data() + index, 0, varSize * sizeof(OutT));
+          for (int i = 0; i < varSize; ++i)
+            (tmpData.data() + index)[i] = std::numeric_limits<Real>::quiet_NaN();
```

Comment:
If I'm reading this right, then you're overriding all the `0` from the `memset` right above, correct?
If yes, is the `memset` still required?

Reply:
Oh, this is debugging code. I will switch back to zeros for sparse. I generally think it is more useful to plot sparse with NaNs so you can see where the sparse fields have been deallocated easily, but I think everyone agreed on filling with zeros at some point.